ble_firmware/ble_fw.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+
+const uint8_t ble_fw_img[] = {
```

Comment:
Zephyr code can't be used in modules.
If required, this code should be hosted in zephyr repo

Reply:
If I remember well I observed this method in hal/infineon. Can you take a look to ```zephyrproject/infineon/cat1cm0p/COMPONENT_CM0P_SLEEP/psoc6_cm0p_sleep.c``` ?

Reply:
> Zephyr code can't be used in modules. If required, this code should be hosted in zephyr repo

You mean maybe under ```zephyr/modules``` ?
I see now that infineon is hosting there the bt stuff for their blobs... :(